client/relay.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "src": "./src",
+  "language": "typescript",
+  "schema": "./src/schema.graphql",
+  "exclude": [
+    "**/node_modules/**",
+    "**/__mocks__/**",
+    "**/__generated__/**"
+  ]
```

Comment:
You can make sure all your relay compiled queries end up in the same folder by adding this:
```
module.exports = {
  ...
  artifactDirectory: "./__generated__",
  ...
}
```